g/go-systemd/github.com_coreos_go-systemd_ubi_8.4.sh

```diff
@@ -0,0 +1,57 @@
+
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package        : github.com/coreos/go-systemd
+# Version        : 48702e0da86bd25e76cfef347e2adeb434a0d0a6
+# Source repo    : https://github.com/coreos/go-systemd
+# Tested on      : UBI 8.4
+# Language      : go
+# Travis-Check  : True
+# Script License : Apache License, Version 2 or later
+# Maintainer     : Vaibhav Bhadade <vaibhav.bhadade@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+PACKAGE_NAME=go-systemd
+PACKAGE_VERSION=${1:-48702e0da86bd25e76cfef347e2adeb434a0d0a6}
+GO_VERSION="go1.17.5"
+
+ 
+#install dependencies
+yum install -y wget git tar dbus gcc-c++&& \
```

Comment:
Looks like, we we need to install `systemd-devel`  package.

Observed below error while running this script on my VM :

```
[root@p006vm76 go-systemd]# go test ./... -v
# github.com/coreos/go-systemd/sdjournal
sdjournal/journal.go:27:11: fatal error: systemd/sd-journal.h: No such file or d                                                                                        irectory
 // #include <systemd/sd-journal.h>
           ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
installation for systemd-devel package added